benchmarks/perf-tool/release-configs/faiss-nsg/test.yml

```diff
@@ -0,0 +1,32 @@
+endpoint: localhost
+test_name: "Faiss NSG Test"
+test_id: "Faiss NSG Test"
+num_runs: 10
+show_runs: false
+steps:
+  - name: delete_index
+    index_name: target_index
+  - name: create_index
+    index_name: target_index
+    index_spec: /home/ec2-user/[PATH]/index.json
+  - name: ingest
+    index_name: target_index
+    field_name: target_field
+    bulk_size: 500
+    dataset_format: hdf5
+    dataset_path: [DATASET_PATH]/sift-128-euclidean.hdf5
+  - name: refresh_index
+  index_name: target_index
+  - name: force_merge
+    index_name: target_index
+    max_num_segments: 10
```

Comment:
I think you should make this value as 1 for best performance.

Reply:
++ LGTM